src/prefer-arrow-functions.spec.ts

```diff
@@ -809,16 +767,15 @@ const invalidWhenAllowNamedFunctions = [
     output: `function foo(a: string): () => string {
       return () => \`bar \${a}\`;
     }`,
-    parser: require.resolve('@typescript-eslint/parser'),
   },
 ];
 
 const ruleTester = new RuleTester({
-  parser: require.resolve('@typescript-eslint/parser'),
-  parserOptions: {
-    ecmaFeatures: { jsx: false },
-    ecmaVersion: 8,
-    sourceType: 'module',
+  // TODO: Remove this `languageOptions` once the tests are updated for flat config:
```

Comment:
Once the tests are updated to support flat config, this can go back to the original values. The new default for `RuleTester` expects flat config tests.